helm/prometheus-rules/templates/alerting-rules/promtail.rules.yml

```diff
@@ -11,10 +11,10 @@ spec:
       rules:
         - alert: PromtailDown
           annotations:
-            description: '{{`Scraping of all promtail pods to check if one failed every 5 minutes.`}}'
+            description: '{{`Scraping of all promtail pods to check if one failed every 30 minutes.`}}'
```

Comment:
Did you check the ksm node metrics? Also, I'm a bit unsure what happens if the container keeps restarting every 2-5 minutes

Reply:
> Did you check the ksm node metrics?

No, what would it be for ?

> Also, I'm a bit unsure what happens if the container keeps restarting every 2-5 minutes

I think we already had this issue with other apps and decided to have 2 different alerts : 1 for when the app is litterally down, and another for when it keeps restarting and/or crashlooping

Reply:
something like that I guess ?

(kube_pod_info * on (cluster_id, pod) group_left() up{container="promtail"}) * on (cluster_id, node) kube_node_status_condition{condition="Ready", status="true"}

Reply:
But let's check with herve :D